build/cdc_data/run.sh

```diff
@@ -64,18 +64,30 @@ fi
 # cd into simple importer dir to run the importer.
 cd $WORKSPACE_DIR/import/simple
 
+# Pick import mode based on value of $SCHEMA_UPDATE_ONLY.
+if [[ $SCHEMA_UPDATE_ONLY == "true" ]]; then
```

Comment:
What are your thoughts on asking users to set the `$MODE` variable themselves vs having separate vars for each mode?

Reply:
I like that that is future-proof - lets us easily add different modes in the future! Updated.